### ISSUE TITLE
LG B4 service menu

### ISSUE THREAD

**Issue by Cowfood1** [Maassoft/ColorControl#455]
Do we know if enabling service menu will work on LG B4 2024 tvs?  Would this be dangerous to try?

**Comment by Cowfood1** [comment]
Right, so my b4 tv is at fw 23.20.24.  I was able to connect without issue Remote worked as it should. However, expert mode did not present the unlock service menu option.  I tried the last three versions of colorcontrol without success.  On the bright side I was able to find GSR and TPC under “config” in the expert menu, the two options I wanted from the service menu.  These seemed to send data to the tv, but whether it took is still in testing here.  So far I think it is where I need it.   But, these options do not show what the current state the tv is in, so I am not 100% sure the settings were applied other than staring at a static image sent from my Mac Studio.  Time will tell if it’s good.

**Comment by Amoeba00** [comment]
I'd be interested to know your results.  I just got a B4 and noticed the same thing, but I didn't make any changes - because I haven't noticed a need to (as compared to my C1).

**Comment by Cowfood1** [comment]
> @Cowfood1 did that work for you? I triggered the command using the `config` menu and hoping that it would work. I am still validating but it looks better anecdotally.

My recording studio ‘where my Mac Studio and B4 monitor are located’ is undergoing upgrades.  So I haven’t been able to do much.  I can say however for many hours of prior use, the dimming has been minuscule -  meaning what I can expect from this hack.  

For now it seems to have done what was needed.
